models/identified_users.sql

```diff
@@ -17,7 +17,9 @@ select
   devices.event_id as last_event_id,
   devices.event_time as last_event_time,
   devices.updated_time as last_updated_time,
-  devices.processed_time as last_processed_time
+  devices.processed_time as last_processed_time,
+  event_properties,
```

Comment:
These columns are ambiguous, do you mean to select them from the `identifies` model or the `devices` model?

We should think through the context of what this means in either case. As an alternative, what are your thoughts on only adding `source_properties` and `event_properties` to the `events` model for now?

Reply:
Good point, it may not achieve the desired result if we add it here. Let me revise things and leave it at the events model for now, that way it's up to the user to then decide if they want to utilise it in another models. Would you agree?